dts/bindings/display/sitronix,st7735r.yaml

```diff
@@ -131,3 +132,14 @@ properties:
       as the inverted value of the RGB pixel-format specified in MADCTL.
       This option is convenient for supporting displays with bugs
       where the actual color is different from the pixel format of MADCTL.
+
+  spi-mode-3wire:
+    type: boolean
+    description: |
+      Enable 3 wire control mode
+
+      3-wire, 9-bit SPI mode. The D/C pin is not used.
+      The 9th SPI bit selects the data or command register.
+
+      4-wire, 8-bit SPI mode.
+      The D/C pin is used to select the data or command register.
```

Comment:
This property should be moved to `dts/bindings/spi/spi-device.yaml` and renamed to `spi-3wire` to be consistent with the Linux kernel.

Reply:
Today I see [st7789v](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/display/sitronix%2Cst7789v.yaml#L21) driver, in this driver this option implicitly handled by "gpio command". 
And MIPI DIB API also does not need additional parameter to select 3/4 wire mode.  Please see: https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/mipi_dbi/Kconfig.spi#L16

So it might be better if we totally remove this additional parameter.

Anyway if you think it will be better  please let me know.